management/telemetry/telemetry-dashboards/telemetry-dashboards.md

```diff
@@ -0,0 +1,197 @@
+transformations, which have consumed most of the credits. The last visual is for the most expensive components. 
+
+#### Configuration Health
+{: .image-popup}
+![Screenshot - Configuration Health](/management/telemetry/telemetry-dashboards/ac-org-usage-config-health.png)
+
+*Configuration Health* gives the health of flows and jobs over time. By hovering over a day, you can see more detailed data (the numbers of flows or jobs on that day that ended with success, error, or 
+warning).
+
+### Project Overview
+This tab provides visuals focused on PPU consumption and shows the most expensive projects, transformations, configurations, and components. It also shows the configurations, projects, and users whose consumption has increased significantly. It can be filtered by branch (development vs. main).
+
+#### PPU and usage
+{: .image-popup}
+![Screenshot - PPU and Usage](/management/telemetry/telemetry-dashboards/ac-project-overview-ppu.png)
+
+These four visuals show PPU consumption from different angles. The first focuses on *PPU per Configuration*, differentiating between SQL, KBC, and CDC credits. *PPU Daily* shows a breakdown of PPU 
+between component types over the period selected. *Component's Usage* is a treemap and provides an overview of how many jobs were executed for the components. You can drill down for a particular 
+configuration or project to see how many credits it consumed. The last visual also shows the credits consumed per component split between SQL, KBC, and CDC credits. 
+
+#### Top 10 
+{: .image-popup}
+![Screenshot - Top 10](/management/telemetry/telemetry-dashboards/ac-project-overview-top10.png)
+
+Top 10 has two tables. The first is the *Top 10 Transformations* based on their consumption. It provides details about the runtime and input and output mapping (e.g., whether any filters were applied or 
+whether the output was incremental or full load). *Top 10 Component Configurations* shows the most expensive configurations and provides information on how long they ran for. 
+
+#### Significant changes (outliers)
+{: .image-popup}
+![Screenshot - Top 10](/management/telemetry/telemetry-dashboards/ac-project-overview-outliers.png)
+
+*Significant changes* shows outliers for projects, configurations, and users. The tables show the average PPU consumption over the previous 30 days for projects, configurations, or users that consumed
+50% more or 50% less on given days. It lists days on which a project billed more than 1 credit. The last column is the percentage of billed credits versus the average. 
+
+### Project Users
+The first section provides an *Overview* of users within a project. They can be filtered by branch type and specific user. It shows the user roles, activities, and how many joined the project 
+in the last seven days. 
+
+#### Overview
```

Comment:
### Project Users
The first section provides an *Overview* of users within a project. They can be filtered by branch type and specific user. It shows the user roles, activities, and how many joined the project 
in the last seven days. 

#### Overview
{: .image-popup}
![Screenshot - Project Users Overview](/management/telemetry/telemetry-dashboards/ac-project-users-overview.png)
The *Users* show total number of users per project with limit, this is the same as for organization usage tab. *Active Users* shows the count of users that did something within the project in the selected period. *Users by Role* in the project based on their role for the selected date range, project and branch type.

#### Activities
{: .image-popup}
![Screenshot - Activities](/management/telemetry/telemetry-dashboards/ac-project-users-activities.png)
The second section, *Activities*, summarizes the users activity in a selected project. How many PPU the users consumed and what was the job status (*UserJobs PPU*) and how active the users were in the project based on the number of changed configurations (*Configuration Updates*). Also it shows newly added project members in the last 7 days with the details (*Newly Added Users Last 7 Days*).